locales/en/messages.json

```diff
@@ -3445,6 +3448,9 @@
     "pidTuningDynamicNotchMinHzHelp": {
         "message": "Defines the lowest dynamic notch center frequency below which the dynamic notch will not go."
     },
+    "pidTuningDynamicNotchMaxHzHelp": {
+        "message": "Defines the highest incoming noise frecuency that is needed to be controlled by the dynamic notch."
+    },
```

Comment:
frequency 🙂

Reply:
Maybe, for max, "Set this to the highest incoming noise frequency that needs to be controlled by the dynamic notch."
If agreeable, for min, "Set this to the lowest incoming noise frequency that needs to be controlled by the dynamic notch."

Reply:
I have updated description and coordinated the new max value for MinHz(250)

Reply:
@ctzsnooze Thanks for explaining. So if I have min at 150 and set max at 200 it will actually be 300?

Reply:
@klutvott123 yes, currently it works as you say.
The FFT has only 32 frequency 'bins' - like the 32 vertical columns in a bar graph.  We need to identify peaks so we need a decent number of columns to work with.  If there aren't many columns, identifying a useful peak gets a bit difficult.  When the max is at least 2x the min, the range of interest will have 16 usable data 'bins' to compare when looking for peaks. It should be easy to find the biggest bin in that range.
If max could be very close to min, there wouldn't, perhaps, be enough bins to work with.  Since the FFT goes looking for peaks, it doesn't matter much how wide the space is.  Previously it was always very wide.
If you have resonance between say 170 and 230, then a suitable range would be 150 to 300.  
It's certainly possible that just 1.5x above minimum might be a suitable range minimum.  There would still be about 10 bins to work with in that case.  Let me know if you think that 2x is too much.

Reply:
I don't think it's a problem having max = 2*min. It's just that there's nothing stopping the user from "breaking the rule". Might be confusing